library/types/src/modules/IP.rb

```diff
@@ -272,32 +241,23 @@ def CheckNetworkShared(network)
     def CheckNetwork4(network)
       generic_check = CheckNetworkShared(network)
       if generic_check != nil
-        return generic_check 
+        return generic_check
 
-        # 192.168.0.1, 0.8.55.999
-      elsif Check4(network)
-        return true 
+      # 192.168.0.0/20, 0.8.55/158
+      elsif network =~ Regexp.new("^[" + @ValidChars4 + "]+/[0-9]+$")
```

Comment:
but only read-only if I remember correctly, so we can make accessor to constant. It is confusing to have editable ValidChars